### ISSUE TITLE
Make a request before authenticated

### ISSUE THREAD

**Issue by abitbul6** [nodogsplash/nodogsplash#210]
Hey,

I want to make an authentication of a user in my server, and send his data to the server. Whats is the best way to it?

I tried to send AJAX request from the splash screen, but because the user didnt authenticated, the request blocked by the nodogsplash.

**Comment by bluewavenet** [comment]
@abitbul6 
I am not sure what you are trying to do but you might like to look at the Forwarding Authentication Service for NDS. https://github.com/nodogsplash/nodogsplash/tree/master/forward_authentication_service

**Comment by abitbul6** [comment]
Hey,
In my splash screen page I do 2 things. I am loading an omage from the web and before the authentication I want to send a request to my server. My server doest have static IP. 
So when I try to make the request (load the image or request to my server), the requests blocked.

Do you have any ideas what to do ?
Thank you

**Comment by bluewavenet** [comment]
Yes, use this, modifying to your requirements:
https://github.com/nodogsplash/nodogsplash/tree/master/forward_authentication_service

**Comment by bluewavenet** [comment]
@abitbul6 
Let me know here if you need more help.
Closing

**Comment by dokaga** [comment]
hi how can in get nodogsplash status from php script

**Comment by dokaga** [comment]
i am having this error when i use shell_exec
ndsctl: nodogsplash probably not started (Error: No such file or directory)

**Comment by bluewavenet** [comment]
@dokaga This means exactly what is says, nodogsplash is not started.
It would be better for you to start a new issue for this where we can check your configuration.
